.gitignore

```diff
@@ -13,6 +13,8 @@ npm-debug.log
 .idea
 .tm_properties
 .settings
+.karma-cache/
+test-bin/
```

Comment:
These are the result of a bad merge and should be removed.

Reply:
I seriously thought I reverted this change. I'll clean this up